packages/eslint/common/no-external-imports/index.js

```diff
@@ -0,0 +1,84 @@
+// require('./typings.d.ts');
+const path = require('path');
+const fs = require('fs');
+
+const DEPENDENCIES_VARIANTS = ['dependencies', 'devDependencies', 'peerDependencies', 'optionalDependencies'];
+
+module.exports = (providedPath, moduleDependencies) => {
+  const dirPath = path.resolve(path.dirname(providedPath));
+  const packageJsonPath = findFilesystemEntity(dirPath, 'package.json');
+  if (typeof packageJsonPath !== 'undefined') {
+    getDependenciesFromPackageJson(packageJsonPath, moduleDependencies);
+  }
+  checkDependenciesInNodeModules(dirPath, moduleDependencies);
+};
+
+function checkDependenciesInNodeModules(currentFolderPath, packageJsonDependencies) {
```

Comment:
It would be good to have `jsdoc` here and for other functions with the brief description what is going on.